Directory.Build.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsPackable>true</IsPackable>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <Title>Test data for $(MSBuildProjectName.Replace('.TestData', '')) tests</Title>
+    <Description>This package provides test data that is used during the execution of the $(MSBuildProjectName.Replace('.TestData', '')) tests.</Description>
+    <Authors>Microsoft</Authors>
+    <Copyright>Copyright © Microsoft Corporation</Copyright>
+    <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</PackageLicenseUrl>
```

Comment:
I think this property is being deprecated, but don't remember 100%.

Reply:
Right it is, but I'm fine with using it at the moment. I quickly tried to find out what to use instead but it wasn't 100% obvious.